apps/pie-docs/src/patterns/overlay-patterns/guidance/guidance.md

```diff
@@ -0,0 +1,386 @@
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/popover/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/popover.svg",
+  alt: "A popover that contains a list of items.",
+  width: "160px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them to display contextual actions.",
+            "They are triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them when there are no actions to perform.",
+            "Don’t use them for large amounts of content or actions."
+        ]
+    }
+} %}
+
+---
+
+## Stacked notification
+
+Stacked notifications allow us to provide users with timely and relevant information or alerts related to certain elements of the interface.
+
+{% notification {
+  type: "information",
+  iconName: "info-circle",
+  message: "Check out our [full documentation](https://pie.design/components/notification/) for this component."
+} %}
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/stacked-notification.svg",
+  mobileSrc:"../../../../assets/img/patterns/overlay-patterns/stacked-notification-narrow.svg",
+  alt: "The compact variant of the notification where the trailing icon and supporting text are on the same line.",
+  width: "320px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them when you want/need information to be attached or dependant of another block of UI.",
+            "Use them only for succinct messaging.",
+            "They are not triggered by the user."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for long form content.",
+            "Don’t use multiple stacked notifications at the same time."
+        ]
+    }
+} %}
+
+---
+
+## System dialogue
+
+A system native dialog is a pre-built element provided by the operating system for displaying alerts, notifications, or messages within an app, adhering to platform design guidelines.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/overlay-patterns/system-dialog.svg",
+  alt: "A system native dialog that contains title, multiple lines of text and two buttons to dismiss or confirm.",
+  width: "275px"
+} %}
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use them only for critical system related information or permissions.",
+            "They can be both triggered by the user or the system."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use them for anything other than critical system messaging or permissions."
+        ]
+    }
+} %}
+
+---
+
+## Toast
+
+A toast show short updates about app processes at the bottom of the screen.
```

Comment:
```suggestion
A toast shows short updates about app processes at the bottom of the screen.
```

Also "app processes" sounds a bit technical yet vague, to me

Reply:
"A toast displays brief updates about the app's actions at the bottom of the screen." mb like this will be better?